root/report/release_label_same_artist.tt

```diff
@@ -0,0 +1,15 @@
+[%- WRAPPER 'layout.tt' title=l('Releases with label the same as the artist') full_width=1 -%]
+
+<h1>[% l('Releases with label the same as the artist') %]</h1>
```

Comment:
Same as with the previous file, "Releases where artist name and label name are the same" might be more clear.